airflow/configuration.py

```diff
@@ -34,23 +34,20 @@
 from collections import OrderedDict
 from configparser import ConfigParser
 
-from .exceptions import AirflowConfigException
+from airflow.exceptions import AirflowConfigException
 
 # show Airflow's deprecation warnings
 warnings.filterwarnings(
     action='default', category=DeprecationWarning, module='airflow')
 warnings.filterwarnings(
     action='default', category=PendingDeprecationWarning, module='airflow')
 
-
-try:
-    from cryptography.fernet import Fernet
-except ImportError:
-    pass
-
-
 def generate_fernet_key():
     try:
+        from cryptography.fernet import Fernet
+    except ImportError:
+        pass
+    try:
         FERNET_KEY = Fernet.generate_key().decode()
```

Comment:
nit: might be better to not use all caps since we're not exporting `FERNET_KEY` into globals. or just return it directly